### DISCUSSION TITLE
Do not remove the error message when user hovers.

### DISCUSSION THREAD

**Discussion by yaswanthmaddula-sureify** [CarlosNZ/json-edit-react#125] [Ideas]
We now have the option to increase the timeout. But i felt its better to apply the timeout after user removes hover if any on the error text. Would be useful if there are a lot of errors in the json.

**Comment by CarlosNZ** [comment]
If there's a lot of errors, I would recommend you handle the error output yourself. There's a [custom `onError` function](https://github.com/CarlosNZ/json-edit-react?tab=readme-ov-file#onerror-function) you can pass in to handle the error UI as you see fit. 

For example, in the [JSON Validation demo](https://carlosnz.github.io/json-edit-react/?data=jsonSchemaValidation), I'm using Toast notifications to display errors, as it's easier to show more detail.
